src/api/endpoint/api/publish.js

```diff
@@ -105,16 +107,15 @@ export default function(router: Router, auth: IAuth, storage: IStorageHandler, c
         });
       });
     };
-
-    if (Object.keys(req.body).length === 1 && isObject(req.body.users)) {
-      // 501 status is more meaningful, but npm doesn't show error message for 5xx
-      return next(ErrorCode.getNotFound('npm star|unstar calls are not implemented'));
+    // star body example: {"_id":"","_rev":"" , users":{}}
+    if (Object.keys(req.body).length === 3 && isObject(req.body.users)) {
```

Comment:
```
Object.keys(req.body).length === 3
```
That's a bit risky, any new npm upgrade might break that. I'd rather do the following, since you really need `_rev` and `users`.

```
const bodyKeys = Object.keys(req.body);
    if (bodyKeys.includes('users') && bodyKeys.includes('_rev') && isObject(req.body.users)) {
      return starApi(req, res, next);
    }
```